packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionStop.ts

```diff
@@ -1,38 +0,0 @@
-/*
```

Comment:
Unless I'm missing something this is still necessary.  From what I recall there are three interactions with function 
`start` -> start the function process 
`invoke` code lens -> run/debug the function
`stop` -> stop the running function

If there's no stop how does the user stop a function that has been started?